ci/lint/06_script.sh

```diff
@@ -6,6 +6,8 @@
 
 export LC_ALL=C
 
+set -ex
```

Comment:
Only downside with this is that CI won't report all lint failures, rather will bail on the first one. Not sure if this is going to be annoying for people. Maybe we could have a `FAIL_FAST` envvar that gates this, and right now is only enabled in the container entrypoint script?

Reply:
Nah, this is a step in the right direction I think.